### PR TITLE
Adding github monorepo script

### PR DESCRIPTION
**Pull request by ajinkya-harness**

### CODE REVIEW

File: catalog-scripts/idp-catalog-wizard-github-mono.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
+import argparse
```
Comment (vikyathharekal) [inline catalog-scripts/idp-catalog-wizard-github-mono.py:1]: Can we rename the file to idp-catalog-wizard-github-monorepo.py? We'll be sharing these files with customers as of now, so better to name it appropriately.

File: catalog-scripts/idp-catalog-wizard-github-mono.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+import requests
+import os
+# import yaml
```
Comment (vikyathharekal) [inline catalog-scripts/idp-catalog-wizard-github-mono.py:4]: We can remove import if not needed

File: catalog-scripts/idp-catalog-wizard-github.py [hunk 3]
```diff
@@ -159,15 +157,17 @@ def parse_arguments():
...
 
 def main():
     args = parse_arguments()
-    
+    global branch
     if not (args.create_yamls or args.register_yamls or args.run_all):
         print("Error: One of --create-yamls, --register-yamls or --run_all must be used.")
         return
-    
+    if args.branch is not None:
```
Comment (vikyathharekal) [inline catalog-scripts/idp-catalog-wizard-github.py:169]: Same comment as above

File: catalog-scripts/idp-catalog-wizard-bitbucket.py [hunk 4]
```diff
@@ -139,15 +137,18 @@ def parse_arguments():
...
 
 def main():
+    global branch
     args = parse_arguments()
     
     if not (args.create_yamls or args.register_yamls or args.run_all):
         print("Error: One of --create-yamls, --register-yamls or --run_all must be used.")
         return
-    
+    if args.branch is not None:
```
[the remark this answers is not in the document]
Comment (ajinkya-harness) [reply catalog-scripts/idp-catalog-wizard-bitbucket.py:150]: We are initialising the global variable with main and then changing it if args is not None

File: catalog-scripts/idp-catalog-wizard-github.py (hunk 3 above)
Comment (ajinkya-harness) [reply catalog-scripts/idp-catalog-wizard-github.py:169]: Same reply as above

**Pull request merged**